keps/sig-node/3675-resource-plugin-manager/README.md

```diff
@@ -0,0 +1,907 @@
+- [ ] (R) Graduation criteria is in place
+  - [ ] (R) [all GA Endpoints](https://github.com/kubernetes/community/pull/1806) must be hit by [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+The authors have taken inspiration from the [CSI development](https://github.com/kubernetes/design-proposals-archive/blob/main/storage/container-storage-interface.md). 
+Kubernetes compute management is tightly integrated with the Kubelet and the 
+existing suite of resources managers including the topology manager, CPU manager, 
+memory manager, and device manager. While these managers have added functionality that 
+has addressed a varied set of use cases, they do present the community with several challenges.
+
+Adding a new capability to one of these managers is slow moving and difficult 
+due to their complex interactions, the level of prudence required given the potential
+impacts, the implied necessity to make the extension work in as many scenarios as
+possible, as well as the overhead heaped onto the sig-node maintainers. More 
+details on the challenges discussed with the community have been captured in 
+the [CPU Management Kubelet Use Cases & Current State document](https://docs.google.com/document/d/1U4jjRR7kw18Rllh-xpAaNTBcPsK5jl48ZAVo7KRqkJk).  
+
+As a result, adding optimizations necessary to support some more niche use cases, 
+to improve total cost of ownership for deployments with more exacting requirements 
+on resources or to provide a mechanism to progressively roll-out the benefits of 
+innovations related to newer CPU architectures is difficult and high cost to community resources.
+
+This proposal aims to address the challenges by introducing a new Container Compute 
+Interface (CCI). The CCI is conceptually equivalent to the CSI in that it will 
+delineate between the responsibility of the Kubelet and lower-level responsibility
+of compute focused plugins for resource assignment capabilities, such CPU and memory. 
+
+We are propsing an iterative implementation approach: in the first stage (alpha) we 
+propose to guarantee the exclusive operation of CCI Manager on cpu and memory resources
+through a feature gate and a requirement to have CPU Manager Policy as None. All pods will be handled 
```

Comment:
Should Memory Manager be disabled as well as CCI Manager is going to handle cpu and memory resources?

Reply:
The alpha version of CCI Manager is CPU only.  We can add that in beta.  Later versions may require both.

Reply:
This is not what I can read here:

> in the first stage (alpha) we propose to guarantee the exclusive operation of CCI Manager on cpu and **memory** resources

Reply:
we will look how to support memory affinity use-cases. We should disable memory manager. We might need time until beta  for complete support but we will start looking into it already now. Minimal target for alpha are the cpu resources.

Reply:
ok , we will limit to cpu and adjust in text